ruby_basic/nikolay.braslavets/home_task_1/6.rb

```diff
@@ -0,0 +1,36 @@
+#For mentors: Not sure about the correct implementation, but it works..
+
+num1 = ARGV[0]
+num2 = ARGV[1]
+operation = ARGV[2]
+
+class Calculator
+  attr_accessor :num1, :num2, :operation
+
+  def initialize()
```

Comment:
The following code is valid.
However since we can omit the parentheses in this case we do just that. They just add visual noise and make the code slightly less readable.
Do not use empty parentheses near method definitions.

Reply:
I misunderstood the documentation correctly, I decided that the initializer without brackets will not work. Fix.